scripts/matlab/AutoscoperConnection.m

```diff
@@ -6,8 +6,10 @@
     end
     methods
         function obj = AutoscoperConnection(address)
-            % Creates a connection to the Autoscoper server
-            obj.socket_descriptor = tcpclient(obj.address, obj.port);
+            % Creates a connection to the Autoscoper 
+            % supports single instance connection only
+            
+            obj.socket_descriptor = tcpip(obj.address, obj.port, 'NetworkRole', 'client');
```

Comment:
@amymmorton `tcpip` will be removed and seems to not be working with MATLAB 2022b, it was replaced with `tcpclient` and `tcpserver`.

Other than that everything else is looking really good!

See [MATLAB docs](https://www.mathworks.com/help/instrument/tcpip.html)

Reply:
I'll take a look- I had version 2020b running and had errors with the tcpclient.  Will confirm